packages/terra-functional-testing/src/commands/utils/downloadScreenshots.js

```diff
@@ -0,0 +1,46 @@
+const axios = require('axios').default;
+const extract = require('extract-zip');
+const fs = require('fs-extra');
+const path = require('path');
+const { Logger } = require('@cerner/terra-cli');
+
+const logger = new Logger({ prefix: '[terra-functional-testing:downloadScreenshots]' });
+
+/**
+ * Downloads and unzips the screenshots from provided url.
+ * @param {string} screenshotUrl - The url to the repository that stores the screenshots.
+ */
+async function downloadScreenshots(screenshotUrl) {
+  if (!screenshotUrl) {
+    logger.warn('A url to the screenshot registry is required to download screenshots.');
+    return;
+  }
+
+  await new Promise((resolve, reject) => {
+    axios({
```

Comment:
does axios support async/await? not a big deal, but we could switch wholesale to that, instead of mixing in `then` statements

Reply:
I use `then` to be consistent with our other projects and also was following the [axios API example](https://github.com/axios/axios#axios-api).

Reply:
well, the axios api also shows an example using async/await 🤷🏼 https://github.com/axios/axios#example